dsf-bpe/dsf-bpe-server/src/main/java/org/highmed/dsf/bpe/listener/CallActivityListener.java

```diff
@@ -0,0 +1,14 @@
+package org.highmed.dsf.bpe.listener;
+
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.camunda.bpm.engine.delegate.ExecutionListener;
+import org.highmed.dsf.bpe.Constants;
+
+public class CallActivityListener implements ExecutionListener
```

Comment:
Consider adding a few lines of documentation, explaining the purpose of this class within the framework.